sandswitches/orms.py

```diff
@@ -148,17 +208,28 @@ class AttrMap(KeyValues):
     }
 
     @property
+    def _subelems(self):
+        return self.elem.xpath('/'.join((self.path, self.tag)))
+
+    @property
     def _attribs(self):
-        path = '/'.join((self.path, self.tag))
-        nodes = self.elem.xpath(path)
-        assert len(nodes) == 1, "Not an AttrMap?"
+        nodes = self._subelems
+        if len(nodes) == 0:  # no sub-elements exists yet
+            raise KeyError
```

Comment:
Yeah this is for sure confusing and must have been from some kind of fix?
I can't recall atm why I did this.